scripts/deploy-tests

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
can we add an extension? `deploy-tests.sh` ?

Reply:
yes, we can.
I didn't do it because cozy-bank didn't do it ! (https://github.com/cozy/cozy-banks/blob/master/scripts/deploy-on-cozy)